integration/integrate-with-splunk.adoc

```diff
@@ -9,6 +9,11 @@ toc::[]
 [role="_abstract"]
 If you are using link:https://www.splunk.com/[Splunk], you can forward alerts from {product-title} to Splunk and view the violations, vulnerability detection, and compliance related data from within Splunk.
 
+[NOTE]
```

Comment:
**suggestion:** I think marking this as **IMPORTANT** would make it stand out. However, this is fine as it is.